apps/web/src/components/Table/ClientManagedTable/types.ts

```diff
@@ -0,0 +1,8 @@
+export type IndividualRow<T> = {
+  original: T;
+};
+
+export type IndividualCell<T> = {
```

Comment:
This is a little strange but I'm guessing you had a good reason.  Why not just `Row` and `Cell`?

Reply:
`react-table` uses `Cell` since it is for a component and I thought it could cause confusion. Also, just copied the exiting one defined in the `PoolTable`

https://github.com/GCTC-NTGC/gc-digital-talent/blob/518b6ee53199eed84530f000fada366eeec9ded1/apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx#L131-L135

However, I am **_totally_** open to changing it if we don't think that would be an issue 👍